src/common/test-util/Cargo.toml

```diff
@@ -5,6 +5,12 @@ edition.workspace = true
 license.workspace = true
 
 [dependencies]
+chrono.workspace = true
+common-config.workspace = true
+futures.workspace = true
 once_cell.workspace = true
 rand.workspace = true
+rskafka.workspace = true
```

Comment:
It's too heavy to add the kafka dependency to the common-test-util. Consider providing it as a new crate or adding these utils to the log-store crate.

Reply:
Some utils are used both by log-store and common-meta. So it cannot be placed into log-store or there will be a cyclic dependency. I will try to move them to a new crate or another more appropriate crate.

Reply:
Some utils are deleted now and `rskafka` is removed from dependencies.